Formula/docbook-xsl.rb

```diff
@@ -61,6 +61,11 @@ def install
   end
 
   def post_install
+    xmlcatalog = "xmlcatalog"
+    on_linux do
+      xmlcatalog = Formula["libxml2"].opt_bin/"xmlcatalog"
```

Comment:
Don't you need a
```ruby
uses_from_macos "libxml2"
```
for this to work? And if you add that, `xmlcatalog` should be in your `PATH` (right?) so this shouldn't be needed.